AppCenterAnalytics/AppCenterAnalytics/Internals/MSAnalytics+Validation.m

```diff
@@ -0,0 +1,58 @@
+#import "AppCenter+Internal.h"
+#import "MSAnalytics+Validation.h"
+#import "MSEventLog.h"
+#import "MSPageLog.h"
+
+// Events values limitations
+static const int minEventNameLength = 1;
+static const int maxEventNameLength = 256;
+
+@implementation MSAnalytics (Validation)
+
+- (BOOL)shouldFilterLog:(id<MSLog>)log {
+  NSObject *logObject = (NSObject *)log;
```

Comment:
I tried to put a break point here but it never got hit. Is the delegate registered?

Reply:
Yeah, this is linker issue in the test app. I've added a workaround such as we use in other categories.